src/core/packages/saved-objects/api-server-internal/src/lib/apis/update.ts

```diff
@@ -178,6 +182,13 @@ export const executeUpdate = async <T>(
 
   // UPSERT CASE START
   if (shouldPerformUpsert) {
+    // Note: Upsert does not support adding accessControl properties. To do so, the `create` API should be used.
```

Comment:
> // Note: Upsert does not support adding accessControl properties. To do so, the `create` API should be used.

Please add the comment to the release note when the time comes.

Reply:
Nit: I wonder is this is really a non-issue now. We support creating objects that support access control without access control metadata (when there is no active user profile). If supported upsert the same way, would this not just passthrough to existing logic?